src/Components/Main.js

```diff
@@ -0,0 +1,31 @@
+import { Component } from 'react';
+import markhor from './img/markhor.jpg';
+import saiga from './img/saiga.jpg';
+
+
+class Main extends Component {
+    render() {
+        
+        return( 
+        <> 
+        <HorenBeast title={'Markhor'} imgURL={'markhor'} description={'Lives in the mountains of central Asia'}/>
+        <HorenBeast title={'Saiga'} imgURL={'saiga'} description={'Native to the dusty steppes of Kazakhstan, Russia'}/>
```

Comment:
Hey there, you are on your way with this!

Take a look at spelling for <HornedBeast> here